### PR TITLE
PulsePoint: Marking cp/ct params to be either String or Int

### PR DESCRIPTION
**Pull request by anand-venkatraman**

### CODE REVIEW

**Review by gargcreation1992 — APPROVED** [review]

File: static/bidder-params/pulsepoint.json [hunk 1]
```diff
@@ -5,11 +5,17 @@
...
+        "string"
+      ],
       "description": "An ID which identifies the publisher selling the impression"
     },
     "ct": {
-      "type": "integer",
+      "type": [
+        "integer",
+        "string"
+      ],
```
Comment (onkarvhanumante) [inline static/bidder-params/pulsepoint.json:18]: @anand-venkatraman  could you add JSON test where `cp` and `ct` are string

[the remark this answers is not in the document]
Comment (onkarvhanumante) [reply static/bidder-params/pulsepoint.json:18]: @anand-venkatraman  requesting to address above comment ^^

[the remark this answers is not in the document]
Comment (anand-venkatraman) [reply static/bidder-params/pulsepoint.json:18]: sorry for the delay, i have added a few tests cases to `validParams` under `params_test.go` file.

[the remark this answers is not in the document]
Comment (onkarvhanumante) [reply static/bidder-params/pulsepoint.json:18]: @anand-venkatraman  thanks for adding `params` test. Requesting to add Json framework tests as well for scenario where `cp` and `ct` are string  in imp.ext

[the remark this answers is not in the document]
Comment (anand-venkatraman) [reply static/bidder-params/pulsepoint.json:18]: addressed in commit https://github.com/prebid/prebid-server/pull/3677/commits/e732c3ff76c15e231754c037d04c08c6e7308fec

**Review by gargcreation1992 — APPROVED** [review]

File: openrtb_ext/imp_pulsepoint.go [hunk 2]
```diff
@@ -4,6 +4,6 @@ package openrtb_ext
 // PubId/TagId are mandatory params
 
 type ExtImpPulsePoint struct {
-	PubID int `json:"cp"`
-	TagID int `json:"ct"`
+	PubID interface{} `json:"cp"`
+	TagID interface{} `json:"ct"`
```
Comment (onkarvhanumante) [inline openrtb_ext/imp_pulsepoint.go:8]: PubID and TagID are expected to be int or string. 
However `interface{}` will allow int, string and any other values 

Should use `StringInt` type
https://github.com/prebid/prebid-server/blob/a1f48b54af8e831927a0f6838f56493fa03efb08/util/jsonutil/stringInt.go#L1-L29

**Review by onkarvhanumante — CHANGES_REQUESTED** [review]
one comment - https://github.com/prebid/prebid-server/pull/3677/files#r1655235030

File: openrtb_ext/imp_pulsepoint.go (hunk 2 above)
Comment (anand-venkatraman) [reply openrtb_ext/imp_pulsepoint.go]: updated params to use `StringInt` instead of `interface{}`

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**